pkg/cmd/cli/describe/describer.go

```diff
@@ -198,7 +199,7 @@ func (d *DeploymentConfigDescriber) Describe(namespace, name string) (string, er
 
 	return tabbedString(func(out *tabwriter.Writer) error {
 		formatMeta(out, deploymentConfig.ObjectMeta)
-		formatString(out, "Latest Version", deploymentConfig.LatestVersion)
+		formatString(out, "Latest Version", strconv.Itoa(deploymentConfig.LatestVersion))
```

Comment:
Latest version 0 is not a version - it means "not deployed".  Make that distinction (and add a test case).